### PR TITLE
Drop cronjob for replication image

### PR DESCRIPTION
**Pull request by nobse**

### CODE REVIEW

File: backup-tools/repl-image/files/replication-run.sh [hunk 1]
```diff
@@ -1,3 +1,5 @@
 #!/bin/bash
-chmod 600 /backup/env/*
-cron -f
+while true; do
+  /usr/local/sbin/backup-replication.sh
+  sleep 14400
```
Comment (Dexus) [inline backup-tools/repl-image/files/replication-run.sh]: 14400 seconds = 4 hours... that would not match our intervals... all 5 mins = 300 seconds where perfekt.. or when need all 10 mins (600)

Comment (nobse) [inline backup-tools/repl-image/files/replication-run.sh]: It's only the replication image that gets changed here, for the backup image there will be another pull request in a few minutes.

Comment (Dexus) [inline backup-tools/repl-image/files/replication-run.sh]: ok
